addon/templates/components/sl-select.hbs

```diff
@@ -13,7 +13,7 @@
 {{/if}}
 
 {{input
-    viewName="sl-select-view"
+    id=inputId
```

Comment:
Thinking "inputId" should at least be "selectId" to keep with the naming convention used by `sl-textarea`, though what about using something more generic instead?

Going with the generic, 2 out of 3 of these components need this functionality, which is identical, so what about moving it into the `sl-input-based` mixin?